application/api/capacityauth/admin.py

```diff
@@ -0,0 +1,46 @@
+# organizations/admin.py
+import typing
+from django.forms import ModelForm
+from django.contrib import admin
+from django.db import models
+from django.http.request import HttpRequest
+from rest_framework_api_key.admin import APIKeyModelAdmin
+from .models import DosUserAPIKey
+
+class DosUserAPIKeyAdminForm(ModelForm):
+    def clean_dosusername(self):
+        #TODO dos user validation
+        return self.cleaned_data["dosusername"] + "-INTERCEPTED"
+
+@admin.register(DosUserAPIKey)
+class DosUserAPIKeyModelAdmin(APIKeyModelAdmin):
+
+    form = DosUserAPIKeyAdminForm
+    is_change_view_call = False
+
+    def get_exclude(self, request, obj=None):
+        if self.exclude == None:
+            self.exclude = ["dosuserid", "name"]
+        else :
+            self.exclude = self.exclude + ["dosuserid", "name"]
```

Comment:
Why not..?

```
if self.exclude:
    self.exclude += ["dosuserid", "name"]  # or perhaps more explicit: self.exclude.extend(["dosuserid", "name", ])
else:
    self.exclude = ["dosuserid", "name"]
```

(FWIW, `None` is to False-y and a list with items in it is Truth-y)

Reply:
Good spot 👍  changed in latest commit